src/composables/layers/layers.composable.ts

```diff
@@ -14,13 +14,9 @@ const themes = useThemes()
 export default function useLayers() {
   function hasIntersect(exclusionA: string, exclusionB: string) {
     try {
-      const concat: number[] = JSON.parse(exclusionA)
-        .concat(JSON.parse(exclusionB))
-        .sort((a: number, b: number) => a - b)
+      const concat = JSON.parse(exclusionA).concat(JSON.parse(exclusionB))
 
-      return concat.some(
-        (element, index, array) => index && element === array[index - 1]
-      )
+      return new Set(concat).size < concat.length
```

Comment:
I don't get how comparing the size of a set can detect an exclusion, what if there is the same length on both layers but not the same ids?

Reply:
concat is the concatenation of both arrays. If there are ids which are in both arrays, they will be removed by the Set creation, so the size will become smaller